src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/0.20.0/pipeline-metadata-entry.xml

```diff
@@ -7,9 +7,6 @@
 
     <changeSet id="pipeline-metadata-entry" author="tom">
 
-        <validCheckSum>7:04d990c0430b5fd7bd63f5bfc462584b</validCheckSum>
-        <validCheckSum>7:fe424670afa75882a10c9178073eecc1</validCheckSum>
```

Comment:
Why are the checksums being removed? Did this move somewhere else? I noticed checksums being removed on other files too.

Reply:
The `validCheckSum` tags were removed as they are no longer required. The previous version of liquibase that we were on had version 7 of the checksum alg (As denoted by the first number before the colon in `7:04d990c0430b5fd7bd63f5bfc462584b`), and the new version has version 8 of the checksum alg.

See here for the block of code where liquibase will check if the checksums are compatible and if not it will nullify them and then set them to the newly calculated value. (https://github.com/liquibase/liquibase/blob/master/liquibase-core/src/main/java/liquibase/changelog/StandardChangeLogHistoryService.java#L250)